pkgs/servers/home-assistant/custom-components/adaptive-lighting/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, buildHomeAssistantComponent
+, home-assistant
+}:
+
+buildHomeAssistantComponent rec {
+  pname = "adaptive-lighting";
```

Comment:
I think the pname should match the domain name.

https://github.com/basnijholt/adaptive-lighting/blob/main/custom_components/adaptive_lighting/manifest.json#L2

Reply:
I have no opinions here - but just from a "how the previous one worked":

`prometheus-sensor` uses the `pname` of `prometheus-sensor` but has it's domain set to `prometheus_sensor` in it's `manifest.json`

https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/home-assistant/custom-components/prometheus-sensor/default.nix#L7

https://github.com/mweinelt/ha-prometheus-sensor/blob/main/custom_components/prometheus_sensor/manifest.json#L2

But will defer to whatever yall decide :)

Reply:
Yeah, if we decide that way the prometheus sensor should be fixed. Not sure yet tbh.

Reply:
I'm working on a fix over in #269940.

Reply:
Please rebase and drop `pname` and instead set `owner` and `domain` instead.

The domain name should be `adaptive_lighting`, same for the attribute name.

Reply:
> Can't we apply normal python normalization here?

No, we'll use canonical values that make sense for this ecosystem.